week1/homework/src/index.js

```diff
@@ -0,0 +1,60 @@
+// import {ContactList} from './contactlist.js';
+const {ContactList}=require('./ContactList');
+import {Contact} from './Contact.js';
+
+const my_contact_list = new  ContactList();
+
+// Can't able to add object doesn't created by contact class
+my_contact_list.addContact({
+    name: "Marco",
+    phone: 42838188
+});
+
+//  Only contacts created with the Contact class are accepted.
+let rafah= new Contact("Rafah",45679908);
```

Comment:
why are you using `let`?

Reply:
you mean should I use const?

Reply:
yeah, try to avoid `let` whenever you can. there's really value (no pun intended) in knowing that a variable will not change value somewhere later